Makefile

```diff
@@ -17,11 +17,12 @@ all: depend generate compile images
 check: depend gofmt vet gometalinter
 
 depend: ## Sync vendor directory by running dep ensure
-	dep version || go get -u github.com/golang/dep/cmd/dep
-	dep ensure -v
+	$$GOPATH/bin/dep version || go get -u github.com/golang/dep/cmd/dep
```

Comment:
This is bad if you don't have `dep` installed in `GOPATH`. Will update tomorrow with a better approach, but at least we found what's the problem.

Reply:
One way to make this better is:

```
export DEP="${GOPATH}/bin/dep"
```

and then use `DEP` wherever needed. That's how we handle for [godep in k/k](https://github.com/kubernetes/kubernetes/blob/e0ce8478b2052a6cef5da6b0bd99457f5bb2c7aa/hack/lib/util.sh#L462).

Reply:
Oops. I also meant to say that we can install it the same way we check for godep in k/k. :)

Reply:
https://github.com/kubernetes/kubernetes/commit/35cdbdf16c9dd9ac9b065684ea4566d6e26e85c2